content/blog/1.0-announcement/index.mdx

```diff
@@ -0,0 +1,73 @@
+
+We’re very proud to announce that over 60 companies, 96 interested parties, and 34 co-authors have brought the best of DevOps into the [OpenGitOps 1.0 Principles and Glossary](https://github.com/open-gitops/documents/releases/tag/v1.0.0).
+Each principle discusses the desired state of a system and how it should be operated.
+
+The desired state of a GitOps managed system must be:
+
+1. **Declarative**
+
+    A [system](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#software-system) managed by GitOps must have its desired state expressed [declaratively](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#declarative-description).
+
+2. **Versioned and Immutable**
+
+    Desired state is [stored](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#state-store) in a way that enforces immutability, versioning and retains a complete version history.
+
+3. **Pulled Automatically**
+
+    Software agents automatically pull the desired state declarations from the source.
+
+4. **Continuously Reconciled**
+
+    Software agents [continuously](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#continuous) observe actual system state and [attempt to apply](https://github.com/open-gitops/documents/blob/v1.0.0/GLOSSARY.md#reconciliation) the desired state.
+
+## GitOps is more than just storing infrastructure-as-code
+
+As [git history shows](https://github.com/open-gitops/documents/commits/v1.0.0), the wording of each principle and linked glossary item was very carefully chosen.
+While many of these principles should be familiar to professionals, many are surprised to learn that they still have work to do to implement GitOps fully.
+
+_Declarative_ is usually the easiest principle to understand.
+Not just infrastructure-as-code but all configuration data that is needed to run a system including the application layer.
+While this doesn’t generally include persistent application data such as database contents it often includes credentials and configuration for data recovery or data access.
+
+_Versioned and Immutable_ is often understood as “use git” but there is more to it than that.
+If that desired state describes using a “latest” tag for example, it is no longer versioned because there is no way to rollback changes in the desired state.
+Likewise, many version control systems can be used in GitOps as long as they meet those two basic requirements and teams use them in a conformant manner.
+
+_Pulled Automatically_ means we have to have software agents constantly observing the desired state.
+While triggers may exist to speed up that observation, a GitOps system shouldn’t exclusively rely on them.
+Software agents need to constantly be aware of what the desired state should be, not only when a deliberate change is made.
+The verb “pull” here is used very clearly in contrast to the way traditional CI/CD pipelines function based on triggers that simply push.
```

Comment:
```suggestion
The verb “pull” here is used very clearly in contrast to the way traditional CI & CD pipelines function based on triggers that simply push.
```
@todaywasawesome @scottrigby @murillodigital I agree with @countspongebob to not conflate CI and CD hence my suggestion here to change "CI/CD" to "CI & CD".

From https://www.youtube.com/watch?v=o2_sGFkYNvA&t=1184s by @countspongebob
> It is really critical to not conflate CI and CD.  I try hard to not use the phrase CI/CD because they are very different practices.  One of the lovely things about the GitOps model is the clean and understandable relationship that is created between CI as a Dev practice, and CD as an Ops practice.  I think the notion of the Immutability Firewall in between them based on Git is really excellent.